bokehjs/src/lib/models/formatters/log_tick_formatter.ts

```diff
@@ -24,6 +25,7 @@ export class LogTickFormatter extends TickFormatter {
   static initClass(): void {
     this.define<LogTickFormatter.Props>({
       ticker: [ p.Instance, null ],
+      use_superscript_exponents:   [ p.Boolean, false   ],
```

Comment:
Since this is going in to 2.0 we could consider enabling this by default. Thoughts @micrenda @mattpap ?

Reply:
I am not very sure if it will be appreciated by all. But, if we should do think correct, we may have to add also to implement it on 'basic_tick_formatter.ts' too, showing something like `2 ✕ 10⁻²⁷`

If you want I can implement it, and we will test it on this branch, to see if it looks good.

Reply:
I'm +1 on enabling it by default. I imagine the heavy majority of users prefer the superscript to the "^2" exponents

Reply:
I'm tentatively +1 as well, I actually think almost all users would prefer it. But I would like to see a survey of how things look with different fonts on different browsers and platforms, so we can make an actual informed decision about what a good default font would be. 

Re: basic tick formatter, I would say let's hold off for now. Certainly if you want to work on it that's great but definitely it should be a separate PR from this one.